src/BolWallet/ViewModels/AddMultiCitizenshipViewModel.cs

```diff
@@ -54,6 +70,20 @@ public async Task Generate()
         }
     }
 
+    public async Task CheckMultiCitizenship()
+    {
+        try
+        {
+            ShortHash = MultiCitizenshipShortHashModel.ShortHash;
+            IsMultiCitizenshipRegistered = await _bolService.IsMultiCitizenship(MultiCitizenshipShortHashModel.CountryCode, ShortHash);
+        }
+        catch (RpcException)
```

Comment:
Maybe add some logging of the error here?

Reply:
done